core/services/chainlink/application.go

```diff
@@ -192,13 +197,76 @@ type ApplicationOpts struct {
 	NewOracleFactoryFn         standardcapabilities.NewOracleFactoryFn
 }
 
+type ApplicationHeartbeat struct {
+	commonservices.Service
+	eng *commonservices.Engine
+
+	beat time.Duration
+}
+
+func NewApplicationHeartbeat(lggr logger.Logger) ApplicationHeartbeat {
+	h := ApplicationHeartbeat{
+		beat: Heartbeat,
+	}
+	h.Service, h.eng = commonservices.Config{
+		Name:  "NodeHeartbeat",
```

Comment:
Would it still be clear if this type/service was called just `HeartBeat`?
```suggestion
func NewHeartbeat(lggr logger.Logger) Heartbeat {
	h := Heartbeat{
		beat: HeartbeatPeriod,
	}
	h.Service, h.eng = commonservices.Config{
		Name:  "Heartbeat",
```

Reply:
Wanted to keep it open to next migrating this to a more generic Heartbeat type in common: https://github.com/smartcontractkit/chainlink-common/pull/976/files

Reply:
I think it would be OK to reuse the name, if the intent is not to use two at the same time :shrug:

Reply:
https://github.com/smartcontractkit/chainlink/pull/15700/commits/74261f48bad52fad3e9578bac8328f8bf0c64498